NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+<packageSources>
+ <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+ <add key="Roslyn nightly builds" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
yeah in scriptcs I just copied the DLLs into the `lib` folder and included them into the created nuget package https://github.com/scriptcs/scriptcs/commit/44f666e82c537fcdde87529eb286c9a7558ae26d

Alternatively, but I'd like to avoid it, we can take Roslyn 3 from myget and publish on nuget under a different name and reference that 😎